apps/bridge/src/utils/hooks/usePrepareERC20Deposit.ts

```diff
@@ -39,7 +40,28 @@ export function usePrepareERC20Deposit({
     ],
     cacheTime: 0,
     staleTime: 1,
-    gas: BigInt(300000),
   });
   return depositConfig;
 }
+
+export async function prepareERC20Deposit({
+  asset,
+  depositAmount,
+  includeTosVersionByte,
+}: UsePrepareERC20DepositProps) {
+  return prepareWriteContract({
+    address: publicRuntimeConfig.l1BridgeProxyAddress,
+    abi: L1StandardBridge,
+    functionName: 'depositERC20',
+    chainId: parseInt(publicRuntimeConfig.l1ChainID),
+    args: [
+      asset.L1contract as Address,
+      asset.L2contract as Address,
+      depositAmount !== ''
+        ? parseUnits(depositAmount, asset.decimals)
+        : parseUnits('0', asset.decimals),
+      100000,
```

Comment:
Any context on why this changed from 30k? Just overshot before?

Reply:
this has been here for a while because wallets were sending too little gas a while ago so we saw a lot of failures. i tested this on mainnet and am no longer seeing the issue though so it appears unnecessary. this is also , for some reason, not enough gas on goerli. so i think we're ok just leaving it out now. we can add it back if we start seeing issues though.